cmd/end2endtest/helpers.go

```diff
@@ -406,13 +494,25 @@ func getCensusParticipantKey(voterAccount *ethereum.SignKeys, censusType string)
 	return key, nil
 }
 
-// matchResult compare the expected vote results base in the overwrite applied, with the actual result returned by the API
-func matchResult(results [][]*types.BigInt, expectedResult [][]string) bool {
-	// only has 1 question
-	for q := range results[0] {
-		if !(expectedResult[0][q] == results[0][q].String()) {
-			return false
+func matchResults(results [][]*types.BigInt, expectedResult [][]string) bool {
+	// iterate over each question to check if the results match with the expected results
+	for i := 0; i < len(results); i++ {
+		for q := range results[i] {
+			if !(expectedResult[i][q] == results[i][q].String()) {
+				return false
+			}
 		}
 	}
 	return true
 }
+
+func sliceIntToStr(intSlice [][]int) [][]string {
```

Comment:
you also can drop this whole func if you make expectedResults a `[][]*types.BigInt`